<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InfoBox</class>
 <widget class="QWidget" name="InfoBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>384</width>
    <height>434</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>20</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(46, 47, 48)</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>350</width>
     <height>131</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="coordinatesLabel">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <property name="text">
       <string>Cordinates: </string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="spTypeLabel">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Spectral Type:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="ID_label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>4</y>
     <width>341</width>
     <height>61</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>410</width>
     <height>3</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #2574F5</string>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="fyllrutaV">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>79</y>
     <width>370</width>
     <height>270</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(46, 47, 48,0.5);
border-radius: 35px;
padding: 5px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="editButton">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>40</y>
     <width>61</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
           border: none;
           border-image: url(:/SpaceKnappar/EditButton(NotPressed).png);
        }
        QPushButton:pressed {
           border-image: url(:/SpaceKnappar/EditButton(Pressed).png);
        }</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="spTypeLineEdit">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>142</y>
     <width>70</width>
     <height>26</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
        border: 1px solid #444;
        border-radius: 10px;
        padding: 5px 10px;
        background-color: #222;
        color: #eee;
    }
    QLineEdit:focus {
        border: 1px solid #2574F5;
    }</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="nameLineEdit">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>240</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
        border: 1px solid #444;
        border-radius: 10px;
        padding: 5px 10px;
        background-color: #222;
        color: #eee;
    }
    QLineEdit:focus {
        border: 1px solid #2574F5;
    }</string>
   </property>
  </widget>
  <widget class="QLabel" name="spTypeLabel_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>91</y>
     <width>160</width>
     <height>129</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Spectral Type:</string>
   </property>
  </widget>
  <zorder>fyllrutaV</zorder>
  <zorder>gridLayoutWidget</zorder>
  <zorder>ID_label</zorder>
  <zorder>line</zorder>
  <zorder>editButton</zorder>
  <zorder>nameLineEdit</zorder>
  <zorder>spTypeLabel_2</zorder>
  <zorder>spTypeLineEdit</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
